### DISCUSSION TITLE
ubuntu 22.04 initial installs python-dev error

### DISCUSSION THREAD

**Discussion by njbowen** [biostars/biostar-handbook-issues#227] [General]
get this python error:
$ sudo apt-get install -y byacc zlib1g-dev python-dev git cmake
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package python-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  python2-dev:i386 python2:i386 python2-dev python2 python-dev-is-python3

**Comment by njbowen** [comment]
used this command:
$ sudo apt install python-is-python3
 and then proceeded with one-stop-shopping install script and all is well. 
even fastq-dump ;-)

**Comment by ialbert** [comment]
Thanks for the note, I made the change. 

These errors are the hardest to catch as we don't usually just get a brand-new system regularly.

A new cohort will run through the installation in September and that will take place soon.
